pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -361,6 +361,12 @@ public Future<Void> sendMessages(final List<? extends Entry> entries, EntryBatch
                 msgOutCounter.add(totalMessages);
                 bytesOutCounter.add(totalBytes);
                 chunkedMessageRate.recordMultipleEvents(totalChunkedMessages, 0);
+            } else {
+                log.warn("[{}-{}] Sent messages to client fail by IO exception[{}], these messages(messages count:"
```

Comment:
the {} in log seems not align to the parameter. IO exception[{}] is the third {}, in the parameter is the last

Reply:
Ah, you are right. already fixed, Thanks

Reply:
Is it better to change the log level to debug? It's a message level log, for a cluster with high consumption throughput, it will introduce too many warning logs. We should be careful to add logs on the message level with `>=` info level.